scripts/evaluate_scispacy_meshtagger.py

```diff
@@ -0,0 +1,40 @@
+"""
+Evaluates SciSpacyMesh tagger against a subset of Mesh
+"""
+from argparse import ArgumentParser
+import pickle
+import json
+
+import pandas as pd
+
+from grants_tagger.scispacy_meshtagger import SciSpacyMeshTagger
+from grants_tagger.utils import load_data
+
+
+def evaluate_scispacy_meshtagger(mesh_label_binarizer_path, mesh_tags_path, mesh_data_path):
+    with open(mesh_label_binarizer_path, "rb") as f:
+        label_binarizer = pickle.loads(f.read())
+
+    disease_mesh_data = pd.read_csv(mesh_tags_path)
+    mesh_name2tag = {name: tag for tag, name in zip(disease_mesh_data['DescriptorUI'],
+                                                    disease_mesh_data['DescriptorName'])}
+
+    mesh_tags_names = label_binarizer.classes_
+    mesh_tags = [mesh_name2tag[name] for name in mesh_tags_names]
+
+    X, Y, _ = load_data(mesh_data_path, label_binarizer)
+    X = X[:10_000]
+    Y = Y[:10_000, :]
```

Comment:
Sure it wont be biased to take the first ones as your subset rather than random?

Reply:
:p you don't like taking the first 10k i sense. i might create a script that creates a sample tbh, it might be useful for comparing against MTI which is yet another tool

Reply:
5428df6c20fb41ac8c57055e98031bddb47305aa and 21bf6e79839b27cb56093540a76ab176a23057ca